lib/msf/core/payload/adapter/fetch/linux_options.rb

```diff
@@ -6,7 +6,8 @@ def initialize(info = {})
         Msf::OptEnum.new('FETCH_COMMAND', [true, 'Command to fetch payload', 'CURL', %w[CURL FTP TFTP TNFTP WGET]]),
         Msf::OptEnum.new('FETCH_FILELESS', [true, 'Attempt to run payload without touching disk by using anonymous handles, requires Linux ≥3.17 (for Python variant also Python ≥3.8','none', ['none','bash','python3.8+']]),
         Msf::OptString.new('FETCH_FILENAME', [ false, 'Name to use on remote system when storing payload; cannot contain spaces or slashes', Rex::Text.rand_text_alpha(rand(8..12))], regex: %r{^[^\s/\\]*$}, conditions: ['FETCH_FILELESS', '==', 'false']),
-        Msf::OptString.new('FETCH_WRITABLE_DIR', [ true, 'Remote writable dir to store payload; cannot contain spaces', '/tmp'], regex: /^\S*$/, conditions: ['FETCH_FILELESS', '==', 'false'])
+        Msf::OptBool.new('FETCH_PIPE', [true, 'Attempt to run payload without touching disk, Linux ≥3.17 only', false]),
```

Comment:
Is this description accurate?
> Attempt to run payload without touching disk, Linux ≥3.17 only

I thought it'd still write the payload to disk depending on the `FETCH_FILELESS` datastore option. Also, what's the limiting factor requiring at least 3.17 for compatibility?

Reply:
Nope; not accurate- I found some things right after I opened this, going to draft it real quick